tests/suites/test_suite_ssl.function

```diff
@@ -3467,9 +3464,7 @@ void ssl_decrypt_non_etm_cbc( int cipher_type, int hash_id, int trunc_hmac,
      * Test record decryption for CBC without EtM, focused on the verification
      * of padding and MAC.
      *
-     * Actually depends on TLS >= 1.0 and either AES, ARIA, Camellia or DES,
-     * but since the test framework doesn't support alternation in
-     * dependency statements, just depend on TLS 1.2 and AES.
+     * Actually depends on TLS 1.2 and AES.
```

Comment:
I think you simplified a bit too much. Older version of TLS were removed, as well as DES ciphersuites, but still, this morally depends on TLS 1.2 and either AES, ARIA or Camellia, but we're choosing to depend just on AES due to lack of support for alternation.